CSS fixes and history updates

fedarko opened the pull request:
- Adjusted `max-width` and `line-height` in the history page, per the sage advice of @DylanLukes.

- Some smaller styling issues:
  - Previously, titles and content(on multiple pages: home page, history, WOS, etc.) wouldn't be centered on small screens. This is fixed now.
  - Added extra space between the links on the home page
  - Similar `max-width` CSS for the home page (albeit without the line height stuff)
  - Added horizontal padding to the history page (makes it look a bit nicer for small screens)

For reference, here are the impacts of these changes on the home page:
| Previous home page | New home page |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4177727/225221165-e1afed49-7ada-42c1-85b8-9213b160a7bb.png) | ![image](https://user-images.githubusercontent.com/4177727/225221218-d3e4d5e1-d2d8-488c-9838-e1ed7988dbc1.png) |


- CSS refactoring:
  - I moved the WOS CSS' title selector (`#title`) to the main `terminal.css` file (`.main-title`), so that pages can use it without needing to load the WOS CSS (like a lot of non-WOS pages were doing before).
  - Trimmed down the "popular" page's CSS to remove code duplication with the WOS CSS.

- Updated the history page:
  - Fixed a few citations (e.g. pointing to different versions of the credits page ([1](https://web.archive.org/web/20050220013616/https://chezbob.ucsd.edu/credits.html), [2](http://cse-gsa.eng.ucsd.edu/grad-student-lounge/chezbob/history)) where applicable).
  - Added some small tidbits of Extra History from the old website.
  - Added some larger tidbits of history (c/o Stefan Savage).

For reference, these screenshots show how this PR impacts the history page:

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4177727/225168928-069c792c-3e0e-47ae-a5d7-09b4a856da58.png) | ![image](https://user-images.githubusercontent.com/4177727/225222099-5eeef5aa-cbf2-48a4-80da-85b3098aab40.png) |

jdeblasio commented:
Can you explain, in words, what was happening, what you changed int he code, and (thus) why you're comfortable this fixes it?

jdeblasio commented:
Oh man. I COULD NOT figure out how this was solving the transfer issue. Derp.

fedarko committed:
Merge branch 'main' of https://github.com/chezbob/chezbob

fedarko committed:
Merge branch 'main' of https://github.com/chezbob/chezbob

fedarko committed:
Adjust max-width and line-height in history CSS

Shoutouts to @DylanLukes :100:

fedarko committed:
Fix two citations

fedarko committed:
EWA shoutout

fedarko committed:
party

fedarko committed:
Fix headers to be centered even on small screens

fedarko committed:
remove CSS code duplication (popular <---> WOS)

I'm not sure if it's kosher to share CSS between "apps" like this,
but the popular CSS was mostly identical to the WOS CSS. To remove the
need for us to manually update multiple places when changing the CSS
(e.g. to fix the title centering issue I just fixed), it's nice to have
this code in one place where possible.

fedarko committed:
Forgot to add back margin-bottom to history.css

fedarko committed:
Run prettier on CSS / HTML / JS; minor content

fedarko committed:
Abstract WOS CSS' #title decl to .main-title; sty

Removes the need for all of these unrelated apps to depend on the WOS
CSS -- now they can just refer to the terminal CSS. This is nice,
because it means that if we need to change the WOS CSS in the future
then it won't have the risk of impacting unrelated stuff. (With the
exception of the "popular" app, which still depends on other stuff
from the WOS CSS, but that's a whole other thing.)

Also, a few other styling things -- added space between <p> links
on the index page, added padding to the history pages, etc.

fedarko committed:
Soda & Espresso history from Stefan

fedarko committed:
extra lore from stefan

fedarko committed:
tense

fedarko committed:
minor ordinary content edits
